src/core/webidl.js

```diff
@@ -234,7 +234,7 @@ function typeIsWhitespace(webIdlType) {
 }
 
 const extenedAttributesLinks = new Map([
-  ["CEReactions", "HTML51#cereactions"],
+  ["CEReactions", "!custom-elements#cereactions"],
```

Comment:
@domenic, sorry, what's the right spec to reference for CEReactions? I see it formally defined in WHATWG HTML without linking to the custom elements spec.